src/fermilib/utils/_jellium_trotter_error.py

```diff
@@ -0,0 +1,461 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Module to compute the second order Trotter error for jellium."""
+from __future__ import absolute_import
+from future.utils import iteritems, itervalues
+from math import sqrt
+
+from fermilib.config import *
+from fermilib.ops import normal_ordered, FermionOperator
+from fermilib.utils._plane_wave_hamiltonian import wigner_seitz_length_scale
+from fermilib.utils import jellium_model, Grid
+
+import numpy
+
+
+def commutator(operator1, operator2):
+    """Return the normal-ordered commutator of two fermionic operators.
+
+    Args:
+        operator1, operator2: FermionOperators.
+    """
+    commutator = operator1 * operator2
+    commutator -= operator2 * operator1
+    commutator = normal_ordered(commutator)
+    return commutator
+
+
+def double_commutator(op1, op2, op3, indices2=None, indices3=None,
+                      is_hopping_operator2=None, is_hopping_operator3=None):
+    """Return the double commutator [op1, [op2, op3]].
+
+    Assumes the operators are from the jellium Hamiltonian.
```

Comment:
First of all, you should reopen this pull request, its a little weird that you close it to make changes. Nobody is going to merge it (or even could merge it if they wanted to) until I completely approve the PR. I will reopen.

Regarding this point, are you saying that you are exploiting some symmetry in the coefficients the jellium Hamiltonian that is not present when there are nuclear charges? If so, you can specialize this function to jellium. But a lot of these functions are totally general so I suggest still putting all this stuff in the trotter_errors module.